### PR TITLE
docs: improve Portuguese translation

### PR DESCRIPTION
**Pull request by gilhrpenner**
Closes #<issue>

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

- Added missing `lang` flag for Netfly page, this was causing to fallback to English when navigating away from that page.
- Slightly improved a few translations

💯

### CODE REVIEW

**Comment by gilhrpenner** [comment]
I added myself to the Portuguese section of code owners if that's ok :)

**Review by Sn0wye — APPROVED** [review]
LGTM! 🎉

**Comment by claudiofreitas** [comment]
Hello, can I be removed from the notifications about Portuguese translations?
Right now I will not have time to help on those for some time.
I will see if I can contribute more in the future :)

**Review by MattFerreira18 — APPROVED** [review]
🚀

**Review by victoriaquasar — APPROVED** [review]

File: .github/workflows/translations.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ jobs:
               fr: ["Defranos", "joachimjusth"],
               no: ["estubmo", "josephayman"],
               pl: ["matibox", "Infiplaya", "PiotrekPKP"],
-              pt: ["claudiofreitas", "minsk-dev", "Sn0wye", "victoriaquasar", "MattFerreira18"],
+              pt: ["claudiofreitas", "minsk-dev", "Sn0wye", "victoriaquasar", "MattFerreira18", "gilhrpenner"],
```
Comment (juliusmarminge) [inline .github/workflows/translations.yml:43]:
```suggestion
              pt: ["minsk-dev", "Sn0wye", "victoriaquasar", "MattFerreira18", "gilhrpenner"],
```

**Review by juliusmarminge — APPROVED** [review]

**Pull request merged**